README.md

```diff
@@ -147,6 +147,62 @@ One (and only one!) of your test files needs to include _snitch_ as:
 
 See the documentation for the [header-only mode](#header-only-build) for more information. This will include the definition of `main()` [unless otherwise specified](#using-your-own-main-function).
 
+## Example build configuration with meson
+
+First, [meson build](https://mesonbuild.com/)
+needs a
+[`subprojects`](https://mesonbuild.com/Subprojects.html#using-a-subproject)
+directory in the project source root
+for dependencies, then:
+
+```bash
+> meson wrap install snitch
+```
+
+downloads a reviewed wrap-file from
+[WrapDB](https://mesonbuild.com/Wrapdb-projects.html)
+to `subprojects/snitch.wrap`  
+Here's an equivalent wrap-git specification, for illustration:
```

Comment:
Is my suggestion correct? I guess people might want to do this to select a very specific commit, or a version that somehow doesn't exist in WrapDB?
```suggestion
Alternatively, you can create your own wrap-file at this location, for example:
```